google/resource_sql_database_instance_test.go

```diff
@@ -629,6 +629,29 @@ func testAccCheckGoogleSqlDatabaseRootUserDoesNotExist(t *testing.T, instance st
 	}
 }
 
+func TestAccSqlDatabaseInstance_PointInTimeRecoveryEnabled(t *testing.T) {
+	t.Parallel()
+
+	masterID := randInt(t)
+
+	vcrTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccSqlDatabaseInstanceDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: fmt.Sprintf(
+					testGoogleSqlDatabaseInstance_PointInTimeRecoveryEnabled, masterID),
```

Comment:
Would you mind also adding another test step (and then another import step) to update the value from true-> false? You can reuse the same config, just make the value of point_in_time_recovery modifiable.

Reply:
added two test steps, with attribute checks during value update and also imports.